site/app/templates/admin/Docker.twig

```diff
@@ -133,10 +134,16 @@
                         <span class="badge badge-{{ capability_to_color[capability] }}">{{capability}}</span>
                     {% endfor %}
                 </td>
+                {% for capability in value %}
+                <td>
+                <button class="send-button btn btn-primary" id="{{ key }}" onclick=confirmationdialog("{{admin_url}}/remove_image",id,'{{capability}}')>Remove</button>
```

Comment:
If you remove the class `send-button` here, the button will then be centered 

![image](https://user-images.githubusercontent.com/12129065/220798608-0ebacce6-f192-4569-8112-04710e57d305.png)

Also I think the `btn-primary`  should be `btn-danger`  or `btn-default` (personally I think `btn-default` is better because its a button most users won't use so better to keep it out of attention) 

Here's some rational behind the different colors
https://submitty.org/developer/software_and_system_design/interface_design_style_guide